.travis.yml

```diff
@@ -22,7 +22,9 @@ before_script:
   - docker pull kubernetes/pause
   - kind load docker-image kubernetes/pause
   - kind get kubeconfig > /tmp/admin.conf
+  - curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && chmod 700 ./get_helm.sh && ./get_helm.sh
 script:
-- mkdir -p ~/gopath/src/sigs.k8s.io/
-- mv ~/gopath/src/github.com/kubernetes-sigs/descheduler ~/gopath/src/sigs.k8s.io/.
-- make test-e2e
+  - mkdir -p ~/gopath/src/sigs.k8s.io/
+  - mv ~/gopath/src/github.com/kubernetes-sigs/descheduler ~/gopath/src/sigs.k8s.io/.
+  - make test-e2e
+  - make lint-chart
```

Comment:
The Travis CI file does not need to be updated. We will use Prow to run the tests, not Travis CI. Sorry if this was not clear in my previous comments.

Reply:
Sorry, I missed this as I was working on the old Travis CI config before I had to rebase my branch.